### ISSUE TITLE
apiserver fails to lookup authorized app and requires restart

### ISSUE THREAD

**Issue by wescarr** [google/exposure-notifications-verification-server#306]
### TL;DR
The `apiserver` somehow gets into a bad state and logs the following error when calling the `verify` endpoint:
```
[35m(github.com/google/exposure-notifications-verification-server/pkg/database/authorized_app.go:181) [0m
[33m[2020-08-21 15:10:38] [0m [31;1m write tcp 169.254.8.1:26730->10.6.0.2:5432: write: connection reset by peer [0m
failed to lookup authorized app
internal error
```

**Expected behavior**
The apiserver to be able to load the authorized app data from the DB reliably

**Observed behavior**
The `/verify` endpoint responds with a `500` error until the service is restarted

### Reproduction
I unfortunately don't have a consistent repro, but I typically create a few verification codes, verify them successfully, and then after a while see the error.

**Screenshots**

**Environment**

- OS: iOS 13.6
- User-Agent: Dart/2.9

The verification services are deployed on GCP Cloud Run

**Additional information**

**Comment by mikehelmick** [comment]
#393 is a candidate fix for this.

**Comment by wescarr** [comment]
@mikehelmick Thanks for following up on this. Will get it deployed shortly and keep an eye out!
